src/App/App.js

```diff
@@ -5,6 +5,7 @@ import Nav from '../Nav/Nav'
 import MovieContainer from '../MovieContainer/MovieContainer'
 import Banner from '../Banner/Banner'
 import MovieDetailView from '../MovieDetailView/MovieDetailView'
+import Error from '../Error/Error'
 
 class App extends Component {
   constructor() {
```

Comment:
Make sure that your initial values in state match the data type they will be come.  SelectedMovie will become an object, so start it as an empty object. This is key for readability and dev empathy